yb-voyager/src/queryparser/query_parser.go

```diff
@@ -72,10 +112,22 @@ func GetObjectTypeAndObjectName(parseTree *pg_query.ParseResult) (string, string
 		}
 		funcNameList := stmt.GetFuncname()
 		return objectType, getFunctionObjectName(funcNameList)
+	case isViewStmt:
+		viewName := viewNode.ViewStmt.View
+		return "VIEW", getObjectNameFromRangeVar(viewName)
+	case IsMviewObject(parseTree):
+		intoMview := createAsNode.CreateTableAsStmt.Into.Rel
+		return "MVIEW", getObjectNameFromRangeVar(intoMview)
 	}
 	return "", ""
 }
 
+func getObjectNameFromRangeVar(obj *pg_query.RangeVar) string {
```

Comment:
What is a rangeVar? Can you add a comment for context?

Reply:
Range var is basically the structure in parser for storing the relation information like relation name, schema name etc.. 
Will add a comment